### PR TITLE
Adding a dependency needed for python-irodsclient

### PR DESCRIPTION
**Pull request by crislawrence**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

@bioconda/core - Please review....my first attempt at any conda recipe.  It is a dependency needed for python-irodsclient.  Thought I'd try this first.

### CODE REVIEW

File: recipes/prettytable/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package:
+  name: prettytable
+  version: "7"
```
Comment (bgruening) [inline recipes/prettytable/meta.yaml]: This should be 0.7, right?

Comment (crislawrence) [inline recipes/prettytable/meta.yaml]: Hopefully 0.7.2.  Python-irodsclient won't work with lesser versions.

Comment (crislawrence) [inline recipes/prettytable/meta.yaml]: Travis-CI has been going for 1:45 min.  The first job finished in just under 10 min.  I really don't know what travis-ci is doing now.  It said that job 2.2 hadn't even started.  So I am at a loss to know what's happening.

**Comment by bgruening** [comment]
Cris, good new and bad news. This package is already in conda-forge ... and bioconda can use the packages from conda-forge. So you can directly submit your irods one and depend on prettytable.
I will close this now, ok?

File: recipes/prettytable/meta.yaml (hunk 1 above)
Comment (crislawrence) [inline recipes/prettytable/meta.yaml]: Actually, I guess the version is too old for me to use.  Let me see if I can create a recipe using the proper version.

Comment (bgruening) [inline recipes/prettytable/meta.yaml]: Wait, I send you a mail :)
Everything is fine here :) Just submit your second PR with the irods-client.

Comment (crislawrence) [inline recipes/prettytable/meta.yaml]: OK....I will use conda-forge and tomorrow, I'll do a PR for python-irodsclient

Comment (bgruening) [inline recipes/prettytable/meta.yaml]: To explain this a little bit. Conda has so called channels - like PPA on Ubuntu. You can mix channels and enable multiple ones. But Conda has a concept of channel priority so you need to enable the channels in a correct order. Every bioconda package has also conda-forge enable. So you can mix bioconda-packages with conda-forge ones.

**Pull request closed without merging**